mne/viz/epochs.py

```diff
@@ -162,6 +163,19 @@ def plot_image_epochs(epochs, picks=None, sigma=0., vmin=None,
     return figs
 
 
+def plot_image_epochs(epochs, picks=None, sigma=0., vmin=None,
+                      vmax=None, colorbar=True, order=None, show=True,
+                      units=None, scalings=None, cmap='RdBu_r', fig=None):
+
+    msg = ('This function has been deprecated for `plot_epochs_image`. '
+           'It will be removed in 0.11')
+    warnings.warn(msg, category=DeprecationWarning)
```

Comment:
don't do this, instead just decorate the function with the `deprecated` decorator